api/helpers/emailHelper.js

```diff
@@ -1,25 +1,22 @@
-const nodemailer = require('nodemailer');
+const MailGun = require('mailgun-js');
 
 const sendEmail = async (address, text, link) => {
-  let testAccount = await nodemailer.createTestAccount();
-
-  let transporter = nodemailer.createTransport({
-    host: 'smtp.ethereal.email',
-    port: 587,
-    secure: false,
-    auth: {
-      user: testAccount.user,
-      pass: testAccount.pass,
-    },
+  let mailGun = new MailGun({
+    apiKey: process.env.EMAIL_API_KEY,
+    domain: process.env.EMAIL_DOMAIN
   });
-
-  return transporter.sendMail({
-    from: '"Fred Foo 👻" <foo@example.com>',
-    to: address, 
-    subject: 'Hello ✔', 
-    text, 
+  
+  let mailOptions = {
+    from: `Beach-resorts-tours <${process.env.EMAIL_USERNAME}>`,
+    to: address,
```

Comment:
`Beach-resorts-tours` this is my project). Our project called ProjectX

Reply:
Done